pipeline/terraform/env.sh.example

```diff
@@ -35,4 +35,37 @@ export TF_VAR_ecs_asg_min_size=2
 export TF_VAR_ecs_asg_max_size=2
 
 # HTTPS certificate ARN from AWS certificate manager
-export TF_VAR_certificate_arn=arn:aws:acm:us-east-1:123412341234:certificate/12341234-3144-4c26-ba3c-123412341234
\ No newline at end of file
+export TF_VAR_certificate_arn=arn:aws:acm:us-east-1:123412341234:certificate/12341234-3144-4c26-ba3c-123412341234
+
+#schema name for the rds-mysql database
+export TF_VAR_identifier=demodb
+
+#database engine specifying "mysql" as the database server of choice in RDS
+export TF_VAR_engine=mysql
+
+#Version of mysql server to launch in RDS
+export TF_VAR_engine_version=5.7.19
+
+# db instance class to launch in RDS
+export TF_VAR_instance_class=db.t2.large
+
+# if you'd like the database encrypted
+export TF_VAR_storage_encrypted=false
```

Comment:
We should consider encrypting, especially if this is a managed service by AWS.

Reply:
will update the value to `true`

Reply:
Thanks.